webapp/src/workspace.ts

```diff
@@ -1217,10 +1268,12 @@ export function resetAsync() {
             if (Cloud.localToken)
                 pxt.storage.setLocal("local_token", Cloud.localToken);
         })
+        .then(() => syncAsync())
+        .then(() => { });
```

Comment:
Do all these header and sync changes make us more robust or give us other advantages?

Reply:
by doing a sync, we update the headers session marker and it will force other tabs to refresh (eventually).